templates/commands/render/action.go

```diff
@@ -141,6 +141,29 @@ func templateAndCompileRegexes(regexes []model.String, scope *common.Scope) ([]*
 	return compiled, merr
 }
 
+// processPaths processes a list of input String paths for go templating, relative paths,
+// OS-specific slashes, and (soon) file globbing.
+func processPaths(paths []model.String, scope *common.Scope) ([]model.String, error) {
+	pathsCopy := make([]model.String, len(paths))
+	copy(pathsCopy, paths)
+
+	var err error
+	for i, p := range pathsCopy {
+		p.Val, err = parseAndExecuteGoTmpl(p.Pos, p.Val, scope)
+		if err != nil {
+			return nil, err
+		}
+		p.Val = filepath.FromSlash(p.Val)
+		p.Val, err = safeRelPath(p.Pos, p.Val)
```

Comment:
I'd be helpful to comment this. Why do we need to call FromSlash before safeRelPath?

Reply:
This is for verifying that safeRelPath processes a path with the OS-specific correct paths, they could be combined into one function as suggested by @drevell but I think it might be ok to keep as-is. What do you think about combining?